src/main/java/org/ow2/proactive/procci/rest/MixinRest.java

```diff
@@ -29,14 +30,17 @@
 
     private final Logger logger = LogManager.getRootLogger();
 
+    @Autowired
+    private CloudAutomationVariables cloudAutomationVariables;
```

Comment:
Is CloudAutomation important in the name? Are there other implementations which hold global variables?
Furthermore, from what I see this class holds variables globally. Either way it should be a service class, not data. Or pure model, not mutable. I guess that variables should be saved in configuration.

Other variables, should only be kept in models which flow with the execution along the stack. Not globally.